Tests/StreamChatUITests/SnapshotTests/CommonViews/PillButton_Tests.swift

```diff
@@ -0,0 +1,112 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+final class PillButton_Tests: XCTestCase {
+    private var subject: PillButton! = .init().withoutAutoresizingMaskConstraints
+
+    // MARK: - Lifecycle
+
+    override func setUp() {
+        super.setUp()
+        subject.setUp()
+        subject.setUpAppearance()
+        subject.setTitle("This text should appear in a pill.", for: .normal)
```

Comment:
The text looks a bit too close to the edges on the screenshots, doesn't it?

Reply:
Good catch, just added a little bit of padding.